gulpfile.js

```diff
@@ -5,14 +5,11 @@ const yargs = require('yargs');
 const browser = require('browser-sync');
 const gulp = require('gulp');
 const panini = require('panini');
-//const rimraf = require('rimraf');
```

Comment:
if you remove the import, you should also delete the commented out use of it in build task.
But before every Build, the dist Folder should be completely wiped

Reply:
I just removed the comment (see `//`)

> But before every Build, the dist Folder should be completely wiped

This is not the case at this moment, this might need to be tackled in another PR?

Reply:
ok for me

Reply:
Moved the issue to https://github.com/corona-warn-app/cwa-website/issues/98